src/Pages/form/Components/NIHSSSection.tsx

```diff
@@ -0,0 +1,121 @@
+import { Box, makeStyles, Typography } from "@material-ui/core";
```

Comment:
It should be extracted to default importing (eg. `import box from "@material-ui/core/box"`).